### ISSUE TITLE
CCE1.1 - no incluye el folio fiscal original, cuando este se captura

### ISSUE THREAD

**Issue by RigoFlores** [jccastillo0007/eFacturaT#974]
Se debe enviar el atributo FolioFiscalOrig, correspondiente al CFDI ESTANDAR, cuando el CFDI que se está timbrando sustituye a otro que fue cancelado.

Esto es, si elaboro un CFDI con cce, luego lo cancelo porque la cagué o por la razón que sea.
El nuevo CFDI que está sustituyendo al anterior, debe incluir el atributo arriba mencionado, correspondiente al UUID original (cancelado). 
Este dato lo capturé en la sección de pago en parcialidades, caja de texto folio,, ya que ahí es donde se captura el atributo en cuestión. (folio fiscal original)

No estoy seguro que tanto se ocupe (respete este criterio), pero de que cancelarán facturas, eso si es del día a día..

**Comment by jccastillo0007** [comment]
ok, esto si no lo tengo listo aún.

**Comment by jccastillo0007** [comment]
Repetido con el ticket 947

**Comment by RigoFlores** [comment]
Not really... aunque uno es 974 y el otro 947... mas allá del orden de los números...
El 974 dice que se debe incluir el folio, tecléandolo incluso.
El 947 pide que se elija de un XML dentro del equipo, para de ahí tomar el uuid.

Hemos avanzado casi nada esta semana... un primer avance para este issue es el 974, y algo mucho mas elegante es el 947...
